packages/utils/parcelforvscode/schemas/parcelrc.schema.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
And this is related to https://github.com/parcel-bundler/parcel/blob/9d6da2c/packages/core/core/src/ParcelConfig.schema.js